Exoft.Gamification/Startup.cs

```diff
@@ -36,11 +80,19 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, IServiceProvider serviceProvider)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
+
+
+                var scope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope();
+                var context = scope.ServiceProvider.GetService<UsersDbContext>();
+
+                context.Database.Migrate();
+
+                ContextInitializer.Initialize(context);
```

Comment:
var scope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope();
                var context = scope.ServiceProvider.GetService<UsersDbContext>();

                context.Database.Migrate();
ContextInitializer.Initialize(context);

please remove this.... Its really bad approach.... You should manually run migrations, and data initialization (you could comment initialization if data)